network-api/networkapi/wagtailcustomization/tests.py

```diff
@@ -7,14 +7,33 @@
 # see https://docs.djangoproject.com/en/3.1/topics/testing/tools/#urlconf-configuration
 @override_settings(STATICFILES_STORAGE="django.contrib.staticfiles.storage.StaticFilesStorage")
 class LocalizedRedirectTests(TestCase):
-    def setUp(self):
+    def test_redirect(self):
+        """Check that we are redirected to the localized version
+        of the homepage when a Redirect object exists.
+        In this example, a Redirect object exists for /test/ -> /
+        so we expect to be redirected to /en/ when we visit /test/
+        """
         redirect = Redirect(old_path="/test", redirect_link="/")
```

Comment:
What do you think about using `/final` as the target here to make it be more similar to the localized test? I find that a bit easier to read for some reason than the plain `/` 😅

Reply:
changed in 7f475ddcb53bc1cc8841fc764343ad68d19affcf